phpcs.xml.dist

```diff
@@ -23,9 +23,24 @@
     <rule ref="WebimpressCodingStandard.PHP.CorrectClassNameCase.Invalid">
         <exclude-pattern>src/Plugin/Hal.php</exclude-pattern>
         <exclude-pattern>test/ResourceFactoryTest.php</exclude-pattern>
```

Comment:
These exclusions shouldn't be necessary - The factories all reference `Interop\ContainerInterface` which is probably `class_aliased` causing the false negative for phpcs, so that's fine, but if an CS error is showing up for these two, it's likely an actual error (?)

Reply:
Thank you @gsteel 

By removing them, I get the following error:

```
Expected class name interop\container\containerinterface; found Interop\Container\ContainerInterface

Expected class name interop\container\exception\containerexception; found Interop\Container\Exception\ContainerException
```
I am not sure why PHPCS requires the class to be lowercase. Is there any other solution instead of ignoring those files?

Reply:
Yes @froschdesign, I did not set the class name to lowercase. Instead, I ignored the error on the PHPCS config file.

So, if you agree, I will keep the ignored errors on the PHPCS config file since we do not want to set those classes to lowercase. Is that ok @froschdesign, @gsteel?

Reply:
Yes - specifically I was talking about the suppressions for Laminas class names - theses should not be suppressed. The class names from Interop should be correctly cased in the code i.e. `Interop\Container\ContainerInterface` and then the invalid error surfaced by PHPCS should be suppressed. The error itself stems from `class_alias` stuff from `laminas-servicemanager` and a possible bug in the related sniff - In theory, if you bump `laminas-servicemanager` to the correct minimum version (^3.11, but preferably ^3.18) where it replaces interop, you can drop Container Interop from deps completely, replace everything with Psr\Container and remove all the suppressions in phpcs.xml - but it's probably best to keep that for another patch as I'm not sure how well that will play with other deps in this project.

Reply:
Thank you. I will apply this change on the next commit.